File: exs/mundo_1/java/016.java
// Crie um programa que leia um número real qualquer pelo teclado e mostre
// na tela a sua porção inteira

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Digite um número real: ");
        double num = scan.nextDouble();
        System.out.println((int) Math.round(num)); // Convert double to int
    }
}

File: exs/mundo_1/ruby/016.rb
=begin
Desafio 016

Problema: Crie um programa que leia um número Real qualquer
          pelo teclado e mostre na tela a sua porção Inteira.

Resolução do problema:
=end

print"Digite um número real: "
n = gets.chomp.to_f

inteiro = n.to_i
puts"A parte inteira de #{n} é #{inteiro}."